src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -976,6 +1011,17 @@ Currently, in order to export a File PFB, \`enableLimitedFilePFBExport\` must be
               const btnConfigs = entry.buttonConfigs;
               const dropdownTitle = entry.dropdownConfig.title;
 
+              const showLimitedFilePFBExportTooltip = (buttonConfig) =>
+                // If limited file PFB export is enabled, PFB export buttons will be disabled
+                // if the user selects multiple files that are on different nodes in the graph.
+                // (See https://github.com/uc-cdis/data-portal/pull/729).
+                // If the user has selected multiple files on different nodes, display a
+                // tooltip explaining that the user can only export files of the same type.
+                // eslint-disable-next-line implicit-arrow-linebreak
```

Comment:
I would prefer not to disable this eslint check, but move those comments outside the arrow

Reply:
Perf, fixed!